### PR TITLE
Add more debug or info log to local cache

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java [hunk 1]
```diff
@@ -76,6 +76,7 @@ public static LocalCacheManager create(AlluxioConfiguration conf) throws IOExcep
     PageStore pageStore = PageStore.create(conf);
     try {
       Collection<PageInfo> pageInfos = pageStore.getPages();
+      LOG.info("Loading {} existing pages", pageInfos.size());
```
Comment (gpang) [inline core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java]: Should this be something like "Creating LocalCacheManager with {} existing pages" ?

File: core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java [hunk 2]
```diff
@@ -143,6 +144,7 @@ private ReadWriteLock getPageLock(PageId pageId) {
 
   @Override
   public boolean put(PageId pageId, byte[] page) throws IOException {
+    LOG.debug("put(pageId={},page={} bytes)", pageId, page.length);
```
Comment (gpang) [inline core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java]: will the `PageId` instance already print out `PageId`? Will `pageId=` be redundant in the log?

**Comment by apc999** [comment]
@gpang  @calvinjia 
PTAL

File: core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java [hunk 3]
```diff
@@ -205,6 +208,7 @@ public boolean put(PageId pageId, byte[] page) throws IOException {
         mEvictor.updateOnPut(pageId);
       }
     }
+    LOG.debug("put({},{} bytes) exits with evicting ({})", pageId, page.length, victimPageInfo);
```
Comment (bf8086) [inline core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java]: Shall we differentiate when there is not enough space and the new page fails to add?

Comment (apc999) [reply core/client/fs/src/main/java/alluxio/client/file/cache/LocalCacheManager.java]: fixed . good catch

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Comment by apc999** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please

**Pull request merged**